app/lib/routes/ctc_domain.rb

```diff
@@ -0,0 +1,9 @@
+module Routes
+  class CtcDomain
+    def matches?(request)
+      app_base_domains = ["localhost", "getyourrefund.org", "staging.getyourrefund.org", "demo.getyourrefund.org"]
+      ctc_hosts = app_base_domains.map { |domain| ["ctc.#{domain}", "www.ctc.#{domain}"] }.flatten
```

Comment:
Does this mean that the ctc domains should work fine out of the box in demo and staging? Should we attempt to push this to staging to validate that?